### PR TITLE
Add ASCII logo and color to the script

### PR DESCRIPTION
**Pull request by ZackeryRSmith**
b74eea902ebc665eede62c4aa80a0e1b5672cd0c - Added an ASCII logo and color using the colorama module

a1c310e3e0b2324ed8520016a34fa1795ef99d94 - Added colorama to `requirements.txt`

3ef43b913b66a5345009989e940191631d639417 - Small bug which messed with cross platfrom support. I forgot to initialize colorama

**Screenshots**
_Windows 10_
image

_Linux (Ubuntu 22.04)_
image

### CODE REVIEW

File: index.py [hunk 1]
```diff
@@ -1,22 +1,41 @@
...
+{Fore.CYAN}##************************#
+{Fore.BLUE}&*************************{Fore.RESET}\n
+""".replace("*", f"{Fore.BLUE}*").replace("(", f"{Fore.GREEN}(")
+
 # inspect.cleandoc() is used to remove the indentation from the message
 # when using triple quotes (makes the code much cleaner)
 # Typicly developers woudln't use cleandoc rather they move the text
 # all the way to the left
-print(inspect.cleandoc("""
+print(logo+inspect.cleandoc(f"""
```
Comment (AlexFlipnote) [inline index.py:30]: Since this is already f-string, might as well be easier to use {logo} inside it?

File: index.py [hunk 2]
```diff
@@ -59,14 +78,15 @@
...
+
     # This dumps our working setting to the config.json file
     # Indent is used to make the file look nice and clean
     # If you don't want to indent, you can remove the indent=2 from code
     json.dump(config, f, indent=2)
 
 
 class FunnyBadge(Client):
-    def __init__(self, *, intents: Intents):
+    def __init__(self, *, intents: Intents) -> None:
```
Comment (AlexFlipnote) [inline index.py:89]: I mean isn't it common knowledge that init never returns anything? Correct me if I am wrong lmao

**Review by AlexFlipnote — CHANGES_REQUESTED** [review]

File: index.py (hunk 1 above)
Comment (ZackeryRSmith) [reply index.py:30]: Caused formatting bugs last time I tried it.

Comment (AlexFlipnote) [reply index.py:30]: Ahh, that makes sense, just add a space like `logo + inspect` then, otherwise flake8 will kill me lmao

File: index.py (hunk 2 above)
Comment (ZackeryRSmith) [reply index.py:89]: Yeah, sorry, that's something my formatter automatically did for me. I can remove it lol

**Review by AlexFlipnote — APPROVED** [review]

**Pull request merged**